src/bio/bio_xstream.c

```diff
@@ -1177,24 +1186,6 @@ alloc_xs_blobstore(void)
 	return bxb;
 }
 
-static inline struct bio_bdev *
-find_sys_bdev(unsigned int role)
```

Comment:
I think all other special handling code for sysdb VOS in BIO and VOS should be reverted too, do you plan to make it in a follow-on patch?